backend/de.metas.business/src/main/java/de/metas/inout/impl/InOutDAO.java

```diff
@@ -421,4 +424,10 @@ public void save(@NonNull final I_M_InOutLine inoutLine)
 	{
 		InterfaceWrapperHelper.saveRecord(inoutLine);
 	}
+
+	@Override
+	public AdTableId getInOutTableId()
```

Comment:
not sure if this method will be needed after doing other suggested changes

Reply:
not needed, removed.